src/models/reportedArticles.js

```diff
@@ -0,0 +1,22 @@
+
+module.exports = (sequelize, DataTypes) => {
+  const reportArticles = sequelize.define('reportedArticles', {
+    reason: {
+      type: DataTypes.STRING,
```

Comment:
Add an id field to the model and set it as PK with type UUIDv4

Reply:
an id field has been added to the reported articles table